index.html

```diff
@@ -1,12 +1,398 @@
 <!DOCTYPE html>
 <html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Девайс</title>
-  </head>
-  <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Девайс</title>
+</head>
+
+<body>
+  <header class="header">
+    <img class="logo" src="" alt="">
+    <form class="user-navigation search-form" action="https://echo.htmlacademy.ru" method="GET">
+      <label class="visually-hidden" for="search">Поле для поиска.</label>
+      <input class="user-navigation search-item search-input" type="text" placeholder="Текст поиска" id="search"
+        name="search">
+      <button class="user-navigation search-item submit-item" type="submit">
+        Найти
+      </button>
+    </form>
+    <ul class="user-navigation-list">
+      <li class="user-navigation-item">
+        <a class="user-navigation-link" href="#">
+          Войти
+        </a>
+      </li>
+      <li class="user-navigation-item">
+        <a class="user-navigation-link" href="#">
+          Сравнить
+        </a>
+      </li>
+      <li class="user-navigation-item">
+        <a class="user-navigation-link" href="#">
+          Корзина
+        </a>
+        <div class="main-navigation-popover visually-hidden">
```

Comment:
```suggestion
        <div class="main-navigation-popover hidden">
```

чем `visually-hidden` отличается от `hidden`?

Reply:
поправь аналогичную ошибку

Reply:
Нам такой разницы не рассказывали вроде, по крайней мере в материалах к 1-ой работе.
Может речь об `overflow` чтобы избыток дочерних элементов прятать?

Или о `display: none`, чтобы весь блок не вытеснял другие элементы?

Нужна подсказка из зала)
![image](https://user-images.githubusercontent.com/52194934/154850372-5cd51720-ab4b-4d13-8803-d33adc3fac6d.png)